felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/OpprettTilbakekrevingRequest.kt

```diff
@@ -30,7 +30,9 @@ data class OpprettTilbakekrevingRequest(
     @field:Valid
     val faktainfo: Faktainfo,
     @field:Valid
-    val institusjon: Institusjon? = null
+    val institusjon: Institusjon? = null,
+    @field:Valid
+    val manuelleBrevmottakere: Set<Brevmottaker> = emptySet()
```

Comment:
Er dette noe som kan settes i tilbakekreving, fra frontend, og ikke være en del av kontraktet? (eller er det allerede mulig?) 
Eller hvorfor må fagsystemet som oppretter en "OpprettTilbakekrevingRequest" sende med brevmottakere? Det er ikke sikkert man har valgt det når OpprettTilbakekrevingRequest blir opprettet hos oss i EF.

Reply:
Det var opprinnelig planen, men for å spare SB for dobbeltarbeid når det dreier seg om de samme, må ba-sak ha mulighet til å sende med brevmottakere opprettet i fagsystemet.
Andre fagsystemer skal ikke trenge å sette noe, siden det er satt en default-verdi der, men ville du foretrukket at den var optional med null som default?

Reply:
Jeg tenkte kanskje litt feil men at den kanskje ikke er satt. Men det viktigste er kanskje at en tom liste her blir håndtert greit i tilbake 👀